respy/likelihood.py

```diff
@@ -465,13 +316,14 @@ def log_like_obs(state_space, data, base_draws_est, tau, optim_paras):
         draws,
         optim_paras["delta"],
         state_space.is_inadmissible[ks],
-        choices,
+        choices.reshape(-1, num_types),
         tau,
     )
 
-    # Multiply the probability of the agent's choice with the probability of wage and
-    # average over all draws to get the probability of the observation.
-    prob_obs = (prob_choices * prob_wages).mean(axis=2)
+    # The resulting prob_choices are actually only smoothed choices per draw and
+    # have to be averaged over draws. They have the shape (nobs, num_types, ndraws)
+    prob_choices = prob_choices.mean(axis=2)
```

Comment:
I think this averaging should be part of `simulate_probability_of_observed_choice`. @tobiasraabe was there a reason for not doing it?

Reply:
Before, ``prob_wages`` had ``n_draws`` as the last dimension like ``prob_choices``. This has probably changed with the Kalman filter, right? Both were combined based on draws and the averaged.

Reply:
Yes it has because the likelihood of the wage does not depend on the draws, so it must have been constant per individual.

Reply:
Then, this can be added to ``simulate_prob...``. I will do it.